src/slack/api.cr

```diff
@@ -11,6 +11,10 @@ class Slack::API
     get_json "/api/channels.list", "channels", Array(Channel)
   end
 
+  def post_message(channel, text)
+    post "/api/chat.postMessage", { "channel" => channel, "text" => text }
```

Comment:
Here you can use `{channel: channel, text: text}` to use a named tuple instead of a hash, so less memory is allocated.

In fact, maybe we can have `post(url, **params)` below, and then you can invoke it like this:

```cr
post "/api/chat.postMessage",  channel: channel, text: text
```